liquidity/liquidity.go

```diff
@@ -600,11 +600,21 @@ func (m *Manager) dispatchBestEasyAutoloopSwap(ctx context.Context) error {
 		return err
 	}
 
-	// Override our current parameters in order to use the const percent
-	// limit of easy-autoloop.
+	// If no fee is set, override our current parameters in order to use the
+	// default percent limit of easy-autoloop.
 	easyParams := m.params
-	easyParams.FeeLimit = &FeePortion{
-		PartsPerMillion: defaultFeePPM,
+
+	switch easyParams.FeeLimit.(type) {
```

Comment:
nit: you can avoid the second type cast below by changing this type switch a bit as explained here: https://go.dev/tour/methods/16